int8 request
---
bool response